### ISSUE TITLE
Stencil should not be inferred when LHS indicies are all absolute

### ISSUE THREAD

**Issue by madgen** [camfort/camfort#18]
```fortran
program absolute_stencil
  integer, dimension(10,20) :: a
  integer :: k

  do k = 1, 10
    a(1,1) = a(k,1)
  end do
end program absolute_stencil
```

A stencil of `stencil readOnce, reflexive(dims=1)` is inferred for `a`. In the case where all LHS indicies are constant, stencil should not be inferred.

**Comment by madgen** [comment]
Already fixed apparently!

**Comment by dorchard** [comment]
Fixed by commit 15ca8ab9c7502ff2ea4760edf1307604aee13199
